src/components/Attachments/AttachmentCarousel/index.tsx

```diff
@@ -229,14 +229,18 @@ function AttachmentCarousel({report, source, onNavigate, setDownloadButtonVisibi
             Gesture.Pan()
                 .enabled(canUseTouchScreen)
                 .onUpdate(({translationX}) => {
-                    if (scale.current !== 1) {
+                    if (!isScrollEnabled.value) {
                         return;
                     }
 
+                    if (translationX !== 0) {
+                        isPagerScrolling.value = true;
+                    }
+
                     scrollTo(scrollRef, page * cellWidth - translationX, 0, false);
                 })
                 .onEnd(({translationX, velocityX}) => {
-                    if (scale.current !== 1) {
+                    if (!isScrollEnabled.value) {
                         return;
                     }
```

Comment:
Fixed. The scroll stuck because we swipe down while scrolling, making `isScrollEnabled` false. Now I update `isPagerScrolling` while scrolling.